x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_panel.styles.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiPanel } from '@elastic/eui';
+import styled from 'styled-components';
+
+export const EuiPanelStyled = styled(EuiPanel)<{ $hasBottomBar: boolean }>`
+  position: fixed;
+  top: 95px;
+  left: 247px;
+  bottom: 0;
+  width: 340px;
+  height: inherit;
+
+  // If the bottom bar is visible add padding to the navigation
+  ${({ $hasBottomBar, theme }) =>
+    $hasBottomBar &&
+    `
+      height: inherit;
+      bottom: 51px;
+      box-shadow:
+        // left
+        -${theme.eui.euiSizeS} 0 ${theme.eui.euiSizeS} -${theme.eui.euiSizeS} rgb(0 0 0 / 15%), 
```

Comment:
I'm surprised you moved this to styled-components instead of keeping with EUI's new css-in-js solution for theme tokens. Eventually these tokens will be removed having to switch back to the way you had it before.

Reply:
Yes, I had to refactor that because I had problems using the `css` interpolation function, it was causing some style rules to get omitted, and not using the `css` function was making tests fail. I changed to the good old way